js/libs/popover.js

```diff
@@ -123,8 +123,8 @@ var $ = require('jquery'),
 
   $(document).on('click', function (e) {
 
-    // Ignore popover and clickover triggers
-    if ($(e.target).attr('data-toggle') === 'popover' || $(e.target).attr('rel') === 'clickover') {
+    // Ignore popover and clickover triggers and children
+    if ($(e.target).attr('data-toggle') === 'popover' || $(e.target).parent().attr('data-toggle') === 'popover' || $(e.target).attr('rel') === 'clickover') {
```

Comment:
Can we assign `$(e.target)` to a variable here? That way we can avoid creating 3 duplicate jQuery objects, bit of a minor performance optimisation, but they all add up

Reply:
👍